doc/dftb+/manual/dftbp.tex

```diff
@@ -209,6 +210,45 @@ \subsection{xyzFormat\{\}}
 }
 \end{verbatim}
 
+\subsection{VaspFormat\{\}}
+\label{sec:dftbp.VaspFormat}
+
+You can also use Vasp's POSCAR or CONTCAR format to specify periodic geometries.
```

Comment:
Suggest adding a link to the format, something like
 You can also use Vasp's \href{https://www.vasp.at/wiki/index.php/POSCAR}{POSCAR} or 
 \href{https://www.vasp.at/wiki/index.php/CONTCAR}{CONTCAR}

Reply:
Added these links.